src/Umbraco.Web.UI.Client/src/views/packages/views/repo.html

```diff
@@ -269,6 +281,10 @@ <h4><strong><localize key="general_searchNoResult">Sorry, we can not find what y
                                     <div class="umb-package-details__information-item-content">{{vm.package.likes}}</div>
                                 </div>
 
+                                <div class="umb-package-details__information-item" ng-if="vm.package.certifiedToWorkOnUmbracoCloud">
+                                    <div class="umb-package-details__information-item-label"><localize key="packager_certifiedToWorkOnUmbracoCloud">Verified to work on Umbraco CLoud</localize></div>
```

Comment:
Hmm here it's `packager_certifiedToWorkOnUmbracoCloud` but I think it should be `packager_verifiedToWorkOnUmbracoCloud`